terraform/main.tf

```diff
@@ -12,7 +12,8 @@ terraform {
 # Base `serverless` IAM support.
 ###############################################################################
 module "serverless" {
-  source = "FormidableLabs/serverless/aws"
+  source  = "FormidableLabs/serverless/aws"
+  version = "0.5.3"
```

Comment:
Should we pin this to the latest `0.6.0`?

Reply:
Yep. I'll update and merge.

Reply:
Done.